code/modules/mob/living/carbon/human/human.dm

```diff
@@ -197,6 +197,35 @@
 				stat("Nation Heir", "[mind.nation.heir ? "[mind.nation.heir]" : "None"]")
 
 
+	//NINJACODE

```

Comment:
This is probably about as snowflakey and non-OOP as it gets for Space Ninja these days. While this still isn't good, this is a longggggggg ways compared to what it was in 2013-2014 (for those who remember).

Reply:
This should probably just do
```
if(istype())
    var/obj/item/clothing/suit/space/space_ninja/SN = wear_suit
    SN.stat_proc(src)
``` or something like that

Reply:
Perhaps some kind of `redirect` component with a `COMSIG_MOBSTAT` signal, that way you don't even need an `istype`

Reply:
could also rig it into the datum_antag system but that might be good for later if you want to split that up.